tenant-isolation-config-service-api/src/main/proto/org/hypertrace/tenantisolation/config/service/v1/tenant_isolation_config_service.proto

```diff
@@ -0,0 +1,44 @@
+syntax = "proto3";
+
+package org.hypertrace.tenantisolation.config.service.v1;
+
+option java_multiple_files = true;
+
+import "org/hypertrace/tenantisolation/config/service/v1/tenant_isolation_group_config.proto";
+
+service TenantIsolationConfigService{
+  rpc GetAllTenantIsolationGroupConfigs(GetAllTenantIsolationGroupConfigsRequest) returns (GetAllTenantIsolationGroupConfigsResponse) {}
```

Comment:
Instead of `GetAll`, just do a plural Get. That can model get all, but it also allows filtered queries if needed (so is easier to evolve over time).